src/game.js

```diff
@@ -0,0 +1,43 @@
+var Game = function(
+  frameOne = new BowlingFrame(),
+  scoreboard = new Scoreboard()
+) {
+  this.frameOne = frameOne;
+  this.scoreboard = scoreboard;
+  this.currentFrame = this.frameOne;
+};
+
+Game.prototype = {
+  readScore: function(frame) {
+    return frame.total();
+  },
+
+  bowl: function(frame) {
+    if (typeof frame.firstRollScore == 'number') {
+      return frame.secondRoll();
+    }
+    var score = frame.firstRoll();
+    this.strikeTracker(frame);
+    return score;
+  },
+
+  strikeTracker: function(frame) {
+    if (frame.firstRollScore === 10) {
+      frame.secondRollScore = 0;
+      return true;
+    }
+    return false;
+  },
+
+  spareTracker: function(frame) {
```

Comment:
Is this function ever use anywhere?  I'm guessing that you had plans to utilise it later, but at the moment I think its just taking up lines :)